pkg/iac/scanners/terraformplan/snapshot/snapshot_test.go

```diff
@@ -0,0 +1,110 @@
+package snapshot
+
+import (
+	"archive/zip"
+	"bytes"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReadSnapshot(t *testing.T) {
+	tests := []struct {
+		name          string
+		dir           string
+		expectedFiles []string
+	}{
+		{
+			name:          "just resource",
```

Comment:
how about reusing `dir` and `name` vars to reduce test code, since they are called the same anyway.

Reply:
done